Automattic/woocommerce-services
# Remove TOS flag: state, condition, and UI

Note: **Not to merge**/release until WCS TOS are consolidated into Jetpack TOS (discussion in p7bbVw-1Rq-p2 and p7bbVw-1Ur-p2)

Removes all TOS functionality that is local to the plugin and independent from Jetpack. As of the WP.com/Jetpack Terms of Service updates accompanying the WooCommerce 3.2 release, there is no need for any additional link or acceptance step.

To test:
- Connect Jetpack (or make sure it's connected)
- Install & activate WCS plugin (or set `tos_accepted` to 0 in `wc_connect_options`)
- Verify that no TOS banner appears, and that plugin is fully initialized & functional
  - e.g. "WooCommerce Services" tab appears on WooCommerce Status page

In case any other implementation complexity comes to mind that was only there to support the TOS check, it may also be worth removing in this PR.

Closes https://github.com/Automattic/woocommerce-services/pull/1138 and https://github.com/Automattic/wp-calypso/issues/16954
